packages/ui-test-utils/src/utils/generateComponentExamples.ts

```diff
@@ -169,22 +172,43 @@ export function generateComponentExamples<Props>(
     page.examples.push(example)
   }
 
-  const maybeAddExample = (props: Props) => {
+  // Serializes the given recursively, faster than JSON.stringify()
+  const fastSerialize = (props: Props) => {
+    const strArr: string[] = []
+    objToString(props, strArr)
+    return strArr.join('')
+  }
+
+  const objToString = (currObject: any, currString: string[]) => {
+    if (!currObject) {
+      return
+    }
+    if (React.isValidElement(currObject)) {
+      currString.push(JSON.stringify(currObject))
+    } else if (typeof currObject === 'object') {
+      for (const [key, value] of Object.entries(currObject)) {
+        currString.push(key)
+        objToString(value, currString)
+      }
+    } else {
+      currString.push(currObject)
+    }
+  }
+
+  const maybeAddExample = (props: Props): void => {
     const componentProps = mergeComponentPropsFromConfig(props)
     const ignore = typeof filter === 'function' ? filter(componentProps) : false
     if (ignore) {
       return
     }
-    // TODO this is wrong, it thinks that objects with object children are the same
-    const propsString = Object.entries(componentProps).sort().toString()
-    //const propsString = JSON.stringify(componentProps)
+    const propsString = fastSerialize(componentProps)
```

Comment:
This is the bugfix. `Object.entries(componentProps).sort().toString()` was just outputting something like `[Symbol]` for JSX nodes in example props. The current solution is faster that simply doing JSON serialization.